app/services/case_remove_pit_extension_service.rb

```diff
@@ -7,12 +7,16 @@ def initialize(user, kase)
     @result = :incomplete
   end
 
+  # TODO (Mohammed Seedat): Setting acting_team: @user.team_for_case(@case)
+  # causes many existing test failures
   def call
     ActiveRecord::Base.transaction do
-      @case.state_machine.remove_pit_extension!(acting_user: @user,
-                                                acting_team: BusinessUnit.dacu_bmt)
-      @case.update external_deadline: (find_original_final_deadline)
-      @case.reload
+      @case.state_machine.remove_pit_extension!(
```

Comment:
Would be good to move the state machine change into the model method remove_pit_deadline!

Reply:
Decided to leave in place pending further technical discussions about current system design